pinot-core/src/main/java/org/apache/pinot/core/util/GapfillUtils.java

```diff
@@ -31,7 +36,10 @@
  */
 public class GapfillUtils {
   private static final String POST_AGGREGATE_GAP_FILL = "postaggregategapfill";
```

Comment:
Is this a left over from previous implementation? I thought `postaggregategapfill` function was no longer being used? I don't see any of the test cases using postaggregategapfill function?

Reply:
PostAggregationGapfillQueriesTest is still using this one. I will deprecate it as part of next PR.

Reply:
ok